internal/pkg/mining/worker.go

```diff
@@ -343,10 +338,25 @@ func (w *DefaultWorker) Mine(ctx context.Context, base block.TipSet, nullBlkCoun
 	return
 }
 
-func (w *DefaultWorker) getPowerTable(ctx context.Context, baseKey block.TipSetKey) (consensus.PowerTableView, error) {
+func (w *DefaultWorker) getPowerTable(baseKey block.TipSetKey) (consensus.PowerTableView, error) {
 	view, err := w.api.PowerStateView(baseKey)
 	if err != nil {
 		return consensus.PowerTableView{}, err
 	}
 	return consensus.NewPowerTableView(view), nil
 }
+
+type ePoStChallengeSeedEntropy struct {
+	// control field for encoding struct as an array
+	_     struct{} `cbor:",toarray"`
+	Epoch abi.ChainEpoch
+	Miner address.Address
+}
+
```

Comment:
Thiis is no longer necessary. 

Given the changes we agreed to wrt passing in the epoch as part of drawing a ticket, we no longer need it here. Only the miner address

Reply:
I was misled by this line, which explicitly mentions epoch for the epost challenge seed, even though all cases already have the epoch mixed in. https://github.com/filecoin-project/specs/pull/883/files#diff-760dd431bb3ed63837eb875d1b328bfcR95